crates/protogen/src/sqlexec/copy_to.rs

```diff
@@ -102,6 +102,9 @@ impl TryFrom<crate::metastore::types::options::CopyToFormatOptions> for CopyToFo
         value: crate::metastore::types::options::CopyToFormatOptions,
     ) -> Result<Self, Self::Error> {
         match value {
+            crate::metastore::types::options::CopyToFormatOptions::Bson => {
+                Ok(CopyToFormatOptions::default())
```

Comment:
Should we be specifying bson specific options here (even if empty)?

Reply:
there aren't any options right now, (I think?) and it feels like we can add this later when we need to, right?

Reply:
Seems reasonable then, I _think_ using default here is fine. If everything passes, then we're handling that fine.

Reply:
eventually it might be nice to have a "write to n partitions" to facilitate parallel table scans, or to be able to write with hive-partitioning, but for now I think having this be "one file output" is good enough.